proxy/http/HttpTunnel.cc

```diff
@@ -1520,11 +1520,15 @@ HttpTunnel::finish_all_internal(HttpTunnelProducer *p, bool chain)
       if (chain == true && c->self_producer) {
         chain_finish_all(c->self_producer);
       }
+      // add fixed code for caching incompleted chunked response case under some terrible network circumstances
+      if (c->vc_type == HT_HTTP_CLIENT && p->chunked_handler.truncation == true) {
+        consumer_handler(VC_EVENT_EOS, c);
```

Comment:
I'm not sure about this. The `HttpSM::tunnel_handler_ua` may not handle the `VC_EVENT_EOS` correctly.

Reply:
I'm sure that HttpSM::tunnel_handler_ua handle VC_EVENT_EOS correctly and it's ok, because I test this case.